irohad/torii/impl/command_service.cpp

```diff
@@ -130,4 +136,117 @@ namespace torii {
     Status(*request, *response);
     return grpc::Status::OK;
   }
+
+  void CommandService::StatusStream(
```

Comment:
General issue: this function looks very large. Better to rework it with subfunctions.